src/main/java/dk/cngroup/lentils/service/TeamService.java

```diff
@@ -17,17 +24,37 @@
     private static final String PIN_CHARACTERS = "0123456789";
 
     private final TeamRepository teamRepository;
+    private final UserRepository userRepository;
+    private final RoleRepository roleRepository;
 
     @Autowired
-    public TeamService(final TeamRepository teamRepository) {
+    private PasswordEncoder passwordEncoder;
+
+    @Autowired
+    public TeamService(final TeamRepository teamRepository,
+                       final UserRepository userRepository,
+                       final RoleRepository roleRepository) {
         this.teamRepository = teamRepository;
+        this.userRepository = userRepository;
```

Comment:
Misto userRepository a roleRepository bych tady dal userService a roleService. Idealni je, kdyz se k repository pristupuje jen v jeji service a vsude jinde uz se pouziva jen ta service. Neni to nutne obecne, ale v projektu jsme se to tak snazili zatim dodrzovat.

Reply:
dobry point - vytahl jsem to a vytvoril pro user a role service :) díky